py-polars/docs/source/reference/testing.rst

```diff
@@ -30,14 +30,14 @@ testing strategies and strategy helpers:
 .. autosummary::
    :toctree: api/
 
-    testing._parametric.dataframes
-    testing._parametric.series
+    testing.parametric.dataframes
+    testing.parametric.series
 
 Strategy helpers
 ~~~~~~~~~~~~~~~~
 
 .. autosummary::
    :toctree: api/
 
-    testing._parametric.column
```

Comment:
I wanted to set this private so that we don't have to give any stability guarantees for this. You can grab the logic if you want, but hey, you are accessing private modules, so it might break.

Reply:
> I wanted to set this private so that we don't have to give any stability guarantees for this. You can grab the logic if you want, but hey, you are accessing private modules, so it might break.

The stability is essentially guaranteed by `tests.parametric.test_testing.py`, and the parametric tests themselves; covers every parameter of every function in there (I'll run a coverage pass to confirm this, and extend if there are any holes). The only way anything would break is if an upgrade to the hypothesis library causes problems (in which case we'd see it in the tests and just not update until it's dealt with). 

We're already using it at work as a powerful addition to the test suite - making it private means we _can't_ formally rely on it, which I think would be a real loss. I'm happy to offer guarantees that I'll take care of this part if anything does actually come up?  ;)

Reply:
> I'll run a coverage pass to confirm this

Just made a small additional patch; parametric testing code now has 99% coverage - only one line not covered (the branch where an environment var would load the `polars.ci_max` profile settings instead of regular `polars.ci`). Added a few % to the `asserts` coverage too while I was at it; that's now at 96%.

Reply:
> We're already using it at work as a powerful addition to the test suite - making it private means we can't formally rely on it, which I think would be a real loss. I'm happy to offer guarantees that I'll take care of this part if anything does actually come up, if that helps? ;)


ok, ok, :)

Reply:
Lol, I know... what can I say, I just love testing! :)
Speaking of which , I see one just failed - will take care of it, hah.